mvc/src/test/java/com/interface21/webmvc/servlet/view/JspViewTest.java

```diff
@@ -0,0 +1,58 @@
+package com.interface21.webmvc.servlet.view;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import jakarta.servlet.RequestDispatcher;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class JspViewTest {
+
+    private HttpServletRequest request;
+    private HttpServletResponse response;
+
+    @BeforeEach
+    void setUp() {
+        request = Mockito.mock(HttpServletRequest.class);
+        response = Mockito.mock(HttpServletResponse.class);
+    }
+
+    @DisplayName("뷰 이름이 redirect:로 시작하면 리다이렉트한다.")
+    @Test
+    void redirect() throws Exception {
+        JspView jspView = new JspView("redirect:/home");
+
+        jspView.render(new HashMap<>(), request, response);
+
+        assertAll(
+                () -> verify(response).sendRedirect("/home"),
```

Comment:
```suggestion
                () -> verify(response).sendRedirect(eq("/home"),
```
Mockito.eq(parameter)를 통해서 정확히 /home이 들어왔는지 검증할 수 있습니다. rawString을 통한 검증은 어떤 방식으로 작동하는지 저도 잘 모르겠네요 😓